pyaedt/icepak.py

```diff
@@ -5276,23 +5176,12 @@ def assign_loss_curve_resistance(self, objects, boundary_name=None, total_power=
         boundary_name : str, optional
             The name of the boundary object that will be created. If not
             provided, a unique name will be generated. Default is ``None``.
```

Comment:
```suggestion
            provided, a unique name is generated. The default is ``None``.
```

Reply:
There are other occurrences of the content in lines 5177 and 5178. Please do a search and replace.